src/compile/compile.ts

```diff
@@ -59,17 +59,26 @@ function assemble(model: Model, topLevelProperties: TopLevelProperties) {
   // autoResize has to be put under autosize
   const {autoResize, ...topLevelProps} = topLevelProperties;
 
+  const encode = model.assembleParentGroupProperties();
+  let width;
```

Comment:
They're not except to extract them from `encode` to be able to pass the remaining properties to the subsequent block. I could have used `delete` as well I guess...